client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx

```diff
@@ -175,29 +184,39 @@ class RegisteredDomainType extends React.Component {
 
 		let message;
 
+		const redemptionCost = this.props.redemptionProduct
+			? formatCurrency(
+					this.props.redemptionProduct.cost,
+					this.props.redemptionProduct.currency_code,
+					{ stripZeros: true }
+			  )
+			: null;
+
 		if ( domain.isRenewable ) {
 			message = translate(
-				'Your domain has expired and is no longer active. You have {{strong}}%(days)s{{/strong}} to renew it at the standard rate before an additional redemption fee is applied. {{domainsLink}}Learn more{{/domainsLink}}',
+				'Your domain has expired and is no longer active. You have {{strong}}%(days)s{{/strong}} to renew it at the standard rate before an additional %(redemptionCost)s redemption fee is applied. {{domainsLink}}Learn more{{/domainsLink}}',
 				{
 					components: {
 						domainsLink,
 						strong: <strong />,
 					},
 					args: {
 						days: moment.utc( domain.renewableUntil ).fromNow( true ),
+						redemptionCost: redemptionCost,
 					},
 				}
 			);
 		} else if ( domain.isRedeemable ) {
 			message = translate(
-				'Your domain has expired and is no longer active. You have {{strong}}%(days)s{{/strong}} to reactivate it during this redemption period before someone else can register it. A additional redemption fee will be added to the price of the domain to reactivate it. {{domainsLink}}Learn more{{/domainsLink}}',
+				'Your domain has expired and is no longer active. You have {{strong}}%(days)s{{/strong}} to reactivate it during this redemption period before someone else can register it. An additional redemption fee of {{strong}}%(redemptionCost)s{{/strong}} will be added to the price of the domain to reactivate it. {{domainsLink}}Learn more{{/domainsLink}}',
 				{
 					components: {
 						domainsLink,
```

Comment:
Could we make these links more context appropriate? For example, for redemptions we have https://en.support.wordpress.com/domains/domain-expiration/#renewing-a-domain-in-the-redemption-period - which took me like three links to find from `/domains` :(

Reply:
Big plus one here! Kamen let me know if you need help tracking down the other links.